downstream.go

```diff
@@ -2304,7 +2304,7 @@ func (dc *downstreamConn) handleMessageRegistered(msg *irc.Message) error {
 		// TODO: support msgid criteria
 		var bounds [2]time.Time
 		bounds[0] = parseChatHistoryBound(boundsStr[0])
-		if bounds[0].IsZero() {
+		if bounds[0].IsZero() && !(subcommand == "LATEST" && boundsStr[0] == "*") {
```

Comment:
Maybe instead of having this special case, we can fill `bounds[0]`? Something among these lines:

```go
if subcommand == "LATEST" && boundsStr[0] == "*" {
    bounds[0] = time.Now()
}
```

With that, all of the logic below can stay as-is.